src/main/resources/asciidoc/css/hibernate.css

```diff
@@ -53,7 +53,7 @@
     --admonition-tip-bg: #f7fff9;
     --admonition-tip: #516c57;
 
-    --code-background: #f3f4f6;
+    --code-background: #f0f8ff7f;
```

Comment:
something didn't go well here 🫣 
I think there should've been 6 chars for the colour. if you were going for the `#f0f8ff`  idk if that's a better choice. The idea was to keep this code background closer to the TOC background, and `#f0f8ff` brings a bit of a blue tint that I don't think we have elsewhere, but maybe I "guessed" the wrong colour 😃

Reply:
8 chars is RGB+alpha, no?

Reply:
ah nice! I always used the `rgba(r, g, b, a)` format when I need the alpha bit changed 🙂